easygems/healpix/__init__.py

```diff
@@ -43,6 +43,14 @@ def isel_extent(dx, extent):
     return np.arange(get_npix(dx))[get_extent_mask(dx, extent)]
 
 
+def broadcast_array(dx, fill_value=np.nan):
+    """Broadcast a limited-area HEALPix array to full shape."""
+    arr = fill_value * np.ones(shape=get_npix(dx))
```

Comment:
This sounds like `np.full` would do a better job, and maybe you want to keep the `dtype`.
```suggestion
    arr = np.full(get_npix(dx), fill_value, dtype=dx.dtype)
```

Reply:
Done 👌